src/lib.rs

```diff
@@ -78,3 +75,8 @@ pub use teloxide_macros::teloxide;
 #[cfg(all(feature = "nightly", doctest))]
 #[doc(include = "../README.md")]
 enum ReadmeDocTests {}
+
+/// A shortcut for `ResponseResult::Ok(val)`.
+pub fn respond<T>(val: T) -> ResponseResult<T> {
+    ResponseResult::Ok(val)
+}
```

Comment:
That's weird, why does the user need to create `ResponseResult`? I thought `ResponseResult` is only expected to be returned from `Bot`'s methods.

Also, isn't `Ok()` shorter?...

Reply:
The reason is that rustc cannot deduce the error type (`E`) in `repl`, for example. We use `respond` in `examples/dices_bot` and the others.

Reply:
~yet another reason why repls are useless~ it's just weird to me to tie up the error type, considering that `Requester::Err` may vary

Reply:
What do you mean by tying up the error type?